docs/api/cookies.md

```diff
@@ -104,7 +104,7 @@ with `callback(error, cookies)` on complete.
 #### `cookies.set(details)`
 
 * `details` Object
-  * `url` String - The url to associate the cookie with.
+  * `url` String - The url to associate the cookie with. An error is thrown if the url is invalid.
```

Comment:
```suggestion
  * `url` String - The url to associate the cookie with. The promise will be rejected if the url is invalid.
```

Reply:
nit: @DeerMichel do you want to align the wording in the documentation that you put on #18751 with this one? I think confirming that it'll be a rejected promise might make the docs _slightly_ clearer 😅

Reply:
sure